drivers/i2c/i2c_common.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Logging of I2C messages
+ *
+ * Copyright Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+
+#include <drivers/i2c.h>
+
+#define LOG_LEVEL CONFIG_I2C_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(i2c);
+
+void i2c_dump_msgs(const char *name, const struct i2c_msg *msgs,
+		   uint8_t num_msgs, uint16_t addr)
+{
+	LOG_DBG("I2C msg: %s, addr=%x", name, addr);
+	for (unsigned int i = 0; i < num_msgs; i++) {
+		const struct i2c_msg *msg = msgs + i;
```

Comment:
I personally prefer this idiom, but the more common one in Zephyr is `&msgs[i]`.  I'm not asking that this change, but would like comments from others as to whether they would.

Reply:
changed